src/uwtools/tests/config/formats/test_base.py

```diff
@@ -127,6 +128,18 @@ def test_dereference(caplog, config):
     assert config == {"foo": 88, "a": 77, "b": {"c": 66}, "d": "{{ X }}"}
 
 
+@pytest.mark.parametrize("fmt2", [FORMAT.ini, FORMAT.nml, FORMAT.sh])
+def test_invalid_config(fmt2, tmp_path):
+    """
+    Test that invalid config files will error when attempting to dump.
+    """
+    fmt1 = FORMAT.yaml
+    outfile = tmp_path / f"test_{fmt1.lower()}to{fmt2.lower()}_dump.{fmt2}"
```

Comment:
Does it matter if these are lowercase? Would they ever not be lowercase?

Reply:
Now that you mention it, its just an arbitrary test object, it doesn't matter at all - I'll take these out